packages/@aws-cdk/aws-apigatewayv2/test/http/api.test.ts

```diff
@@ -220,4 +220,28 @@ describe('HttpApi', () => {
       expect(countMetric.statistic).toEqual(statistic);
     });
   });
+
+  test('description is set', () => {
+    const stack = new Stack();
+    const api = new HttpApi(stack, 'api', {
+      description: 'My Api',
+    });
+
+    expect(stack).toHaveResource('AWS::ApiGatewayV2::Api', {
+      Name: 'api',
+      ProtocolType: 'HTTP',
+      Description: 'My Api',
+    });
+
+    expect(stack).toHaveResource('AWS::ApiGatewayV2::Stage', {
+      ApiId: stack.resolve(api.httpApiId),
+      StageName: '$default',
+      AutoDeploy: true,
+    });
+
+    expect(stack).not.toHaveResource('AWS::ApiGatewayV2::Route');
+    expect(stack).not.toHaveResource('AWS::ApiGatewayV2::Integration');
+
+    expect(api.url).toBeDefined();
```

Comment:
Just the one assertion is sufficient since you're only checking that the description is set.

```suggestion
    expect(stack).toHaveResource('AWS::ApiGatewayV2::Api', {
      Name: 'api',
      ProtocolType: 'HTTP',
      Description: 'My Api',
    });
```

Reply:
Makes sense 🚀